Composer/packages/client/src/recoilModel/DispatcherWrapper.tsx

```diff
@@ -86,7 +86,8 @@ const InitDispatcher = ({ onLoad }) => {
   //so use the useLayoutEffect here
   useLayoutEffect(() => {
     setDispatcher(dispatcherRef.current);
-    prepareAxiosWithRecoil(currentDispatcherState);
+    prepareAxiosWithRecoil();
+    currentDispatcherState; // ToDo: deprecate this
```

Comment:
Consider providing more details about when we will deprecate this.

Reply:
There is a wider swathe of code that dates to a previous authentication scheme related that never shipped. I was going to open another ticket to track a separate "clean up old unused auth code" task....

I *think* I can probably go back and remove that parameter up the chain